mil_perception/src/mil_vision_lib/cv_tools.cpp

```diff
@@ -0,0 +1,70 @@
+#include <mil_vision_lib/cv_tools.hpp>
+
+bool mil_vision::get3DPointsFromParam(std::string topic, std::vector<cv::Point3f>& points)
+{
+    XmlRpc::XmlRpcValue list;
+    if (!ros::param::get(topic, list)) return false;
+    return mil_vision::get3DPointsFromParam(list, points);
+}
+bool mil_vision::get3DPointsFromParam(ros::NodeHandle& nh, std::string topic, std::vector<cv::Point3f>& points)
+{
+    XmlRpc::XmlRpcValue list;
+    if (!nh.getParam(topic, list)) return false;
+    return mil_vision::get3DPointsFromParam(list, points);   
+}
+bool mil_vision::get3DPointsFromParam(XmlRpc::XmlRpcValue& list, std::vector<cv::Point3f>& points)
```

Comment:
I'd say that this is an abuse of overloading since the functions are semantically different. This is not really recursive although the naming could lead one to blelieve so at a glance.